modules/stir_shaken/stir_shaken.c

```diff
@@ -814,6 +814,9 @@ static int add_identity_hf(struct sip_msg *msg, EVP_PKEY *pkey,
 	hdr_buf.len += cr_url->len;
 	hdr_buf.s[hdr_buf.len++] = '>';
 	hdr_buf.s[hdr_buf.len++] = ';';
+	memcpy(hdr_buf.s + hdr_buf.len, HDR_ALG_PARAM_S, HDR_ALG_PARAM_LEN);
++       hdr_buf.len += HDR_ALG_PARAM_LEN;
++       hdr_buf.s[hdr_buf.len++] = ';';
```

Comment:
Can you remove the '+' characters at the start of these lines?

Reply:
I have removed the '+' characters and build checks are running now.